src/api/users.coffee

```diff
@@ -42,16 +43,91 @@ exports.authenticate = (email) ->
     utils.logAndSetResponse this, 500, "Error during authentication #{e}", 'error'
 
 
+
+#################################
+### Reset password Functions ###
+#################################
+
+
+passwordResetPlainMessageTemplate = (firstname, setPasswordLink) -> """
+<---------- Existing User - Reset Password ---------->
+Hi #{firstname},
+
+A request has been made to reset your password on OpenHIM
```

Comment:
Please add details of which him instance, i.e. "A request has been made to reset your password on the OpenHIM instance running on X". The detail is already stored in config (e.g. see [alerting](https://github.com/jembi/openhim-core-js/blob/35c3f4d878c978bfb8483c969c83c964c8128f7b/src/alerts.coffee#L22)).